lib/HeroBonus.cpp

```diff
@@ -75,9 +75,8 @@ const std::map<std::string, TLimiterPtr> bonusLimiterMap =
 	{"DRAGON_NATURE", std::make_shared<HasAnotherBonusLimiter>(Bonus::DRAGON_NATURE)},
 	{"IS_UNDEAD", std::make_shared<HasAnotherBonusLimiter>(Bonus::UNDEAD)},
 	{"CREATURE_NATIVE_TERRAIN", std::make_shared<CreatureTerrainLimiter>()},
-	{"CREATURE_FACTION", std::make_shared<CreatureFactionLimiter>()},
+	{"CREATURES_ONLY", std::make_shared<CreatureLevelLimiter>()},

```

Comment:
Why some limiters were removed even though they are still present in code?

Reply:
Because that limiters will not be functional if initialized by one string without a parameters

Reply:
Ah, so this is code for parameter-less initialization, while code that provides initialization with parameters still accepts these limiters?

OK then.